tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,25 @@ def test_removing_not_found_returns_none():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    #raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert result is None 
```

Comment:
Nice assert, we definitely want to confirm our return value is `None`. We generally also want to confirm that there weren't side affects or changes to the input that we didn't intend. What could we check about the vendor's inventory to confirm that its content didn’t change?

Reply:
We could confirm the size of the vendor's inventory to make sure it didn't get changed. 
(eg. assert len(vendor.inventory == 3) 
We could also check if all items we originally added to the vendor's inventory still exist after the function call. 
(eg. assert "a" in vendor.inventory
        assert "b" in vendor.inventory
        assert "c" in vendor.inventory)

Reply:
Nice, those would be great to add!